docs/conf.py

```diff
@@ -30,19 +30,14 @@
 from docutils.utils import get_source_line
 
 
-def _warn_node(self, msg, node):
-    """Do not warn on external images."""
-    if not msg.startswith('nonlocal image URI found:'):
-        self._warnfunc(msg, '{0}:{1}'.format(get_source_line(node)))
-
-sphinx.environment.BuildEnvironment.warn_node = _warn_node
-
-
 # -- General configuration ------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
 #needs_sphinx = '1.0'
 
+# Do not warn on external images.
```

Comment:
@lnielsen ... and by keeping an open-ended sphinx requirement, we'll be in the same position in 3-4 weeks or months again.  https://github.com/inveniosoftware/cookiecutter-invenio-module/commit/c8d93e4f06be8f329bb25ea69c19a53ffbcc980a#commitcomment-17784747